resources/remote-environments/templates/deployment.yaml

```diff
@@ -102,4 +102,4 @@ spec:
           periodSeconds: 10
         ports:
         - containerPort: {{ .Values.eventService.deployment.args.externalAPIPort }}
-          name: http-api-port
+          name: ext-api-port
```

Comment:
Hi @marcobebway,

Both Event service and Metadata service are exposed over Nginx-Ingress. Therefore, I didn't care for the Istio specific naming convention. But your point is valid, and it might be an issue with tracing. I will apply a change in the opposite direction.